xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/javascript.vm

```diff
@@ -77,10 +78,26 @@
   'bundles': {},
   'config': {}
 })
+#set ($momentRelativePath = 'min/moment.min')
+#set ($momentBasePath =
+  $stringtool.substringBeforeLast($services.webjars.url('momentjs', 'min/moment.min'), $momentRelativePath))
+## We have to declare momentjs as a RequireJS package in order to be able to load momentjs locales on demand using
+## RequireJS. See https://github.com/requirejs/requirejs/issues/1554 .
+#set ($requireConfig.packages = [{
```

Comment:
Note that I just added a ``packages`` section above inside the RequireJS configuration so you should put this above too. Do a git pull first.